### ISSUE TITLE
Interrupting currently playing sound when starting a new sound

### ISSUE THREAD

**Issue by lachsen** [markokajzer/discord-soundbot#51]
I just wanted to request a certain feature that would be very nice to have: being able to interrupt the currently playing sound when starting a new sound.

In our context we very often have multiple people starting sounds in parallel that usually works much better when sounds are interrupting each other - especially considering some of our sounds go up to 30-60 seconds long.

**Comment by markokajzer** [comment]
Hey again! Thank you for the feature suggestion, I'll look into it. I also realized that there is no possibility to skip the current sound, so might add that as well while at it 🤔

**Comment by markokajzer** [comment]
There are two new commands on master that might be what you are looking for.

Use `!skip` to skip the currently playing sound.
Use `!next <sound>` to interrupt the currently playing sound with a new one while persisting the queue.

Try it out and let me know what you think ❤️If you have any more feedback, feel free to reach out whenever!
